pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -297,6 +298,77 @@ func (r *ReconcileStorageCluster) newStorageClasses(initData *ocsv1.StorageClust
 	return ret, nil
 }
 
+// newRBDSnapshotClass returns a SnapshotClass for RBD backing
+func (r *ReconcileStorageCluster) newRBDSnapshotClass(initData *ocsv1.StorageCluster) *snapapi.VolumeSnapshotClass {
+	retSC := &snapapi.VolumeSnapshotClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: generateNameForRBDSnapshotClass(initData),
+		},
+		Driver: "rook-ceph.rbd.csi.ceph.com",
```

Comment:
the driver name works wrong, it should be `<namespace>.rbd.csi.ceph.com`

Reply:
Thanks Madhu... Added the changes...